Dockerfile

```diff
@@ -2,7 +2,6 @@
 FROM quay.io/giantswarm/golang:1.14.7 AS gsctl
 RUN git clone https://github.com/giantswarm/gsctl.git
 WORKDIR /go/gsctl
-RUN git checkout add-json-output
```

Comment:
IMO we can skip the git clone and build altogether -- just pull the gsctl binary

Reply:
with `go install`? The current changes in gsctl master are not in any release yet

Reply:
I reckon this will work until gsctl is released with the json changes.